.travis.yml

```diff
@@ -43,10 +43,20 @@ cache:
   directories:
     - node_modules
 
-before_install:
-
 before_script:
   - npm install -g gemini@5.2.0 html-reporter@2.0.1
 
 script:
   - node scripts/travis
+
+jobs:
+  include:
+    - stage: publish
+      if: (NOT type IN (pull_request)) AND branch = master
```

Comment:
Would we rather it be `if : type = push AND branch = master`? We only want to have this run on a code push to master right? The other types are api/cron, so we have a cron job doing daily testing now that would probably trigger this as well.

Reply:
I'll test with your suggestion and  `if: (NOT type IN (api, cron, pull_request)) AND branch = master`

I'm not 100% sure if github does a push when you merge with the PR button from the web UI and I'm not seeing anything searching the docs. All I see is this
```
Pull requests are merged using the --no-ff option, except for pull requests with squashed or rebased commits, which are merged using the fast-forward option.
```

The intent here is to only run when changes are merged into the master branch.